Gems/Atom/RHI/Code/Include/Atom/RHI/DrawPacketBuilder.h

```diff
@@ -74,6 +74,8 @@ namespace AZ
 
             const DrawPacket* End();
 
+            const DrawPacket* Clone(const DrawPacket* original);
```

Comment:
Let's add a comment here to be more explicit about the DrawSrg, since I think that might surprise someone at some point.
//! Make a copy of an existing DrawPacket.
//! Note: the copy will reference the same DrawSrg as the original, so it is not possible to vary the DrawSrg values between the original draw packet and the cloned one.
//! Only settings that can be modified via the DrawPacket interface can be changed after cloning, such as SetRootConstant and SetInstanceCount